skietbaan_fe/src/bootstrap/NavbarMenuUserStyles.css

```diff
@@ -1,172 +1,203 @@
-@media only screen and (min-width: 580px) {
-  .dropdown-menu.show {
-    background-color: #f3f4f9;
+@media only screen and (min-width: 576px){
+  .dropdown-menu.show{
+    background-color: #F3F4F9;
     border: none;
-    border: 1px solid #f3f4f9;
+    border: 1px solid  #F3F4F9;
+    
   }
 
-  .leaderboard-icon-grey {
-    height: 40px;
-    width: 59px;
+  .profile-icon-grey{
+    height: 27px;
+    width: 28px;
   }
 
-  .add-score-icon-grey {
-    height: 40px;
-    width: 49px;
+  .leaderboard-icon-grey{
```

Comment:
`.leaderboard-icon-grey` and 
`.add-score-icon-grey` and 
`.create-icon-grey` are the same

Reply:
made a generic one